spring-cloud-dataflow-server-core/src/main/resources/org/springframework/cloud/dataflow/server/db/migration/mysql/V1__INITIAL_SETUP.sql

```diff
@@ -0,0 +1,183 @@
+create table app_registration (
+  id bigint not null,
+  object_version bigint,
+  default_version bit,
+  metadata_uri longtext,
+  name varchar(255),
+  type integer,
+  uri longtext,
+  version varchar(255),
+  primary key (id)
+);
+
+create table audit_records (
+  id bigint not null,
+  audit_action bigint,
+  audit_data longtext,
+  audit_operation bigint,
+  correlation_id varchar(255),
+  created_by varchar(255),
+  created_on datetime,
+  primary key (id)
+);
+
+create table if not exists hibernate_sequence (
+  next_val bigint
+);
+
+insert into hibernate_sequence values ( 1 );
+
+create table stream_definitions (
+  definition_name varchar(255) not null,
+  definition longtext,
+  primary key (definition_name)
+);
+
+create table task_definitions (
+  definition_name varchar(255) not null,
+  definition longtext,
+  primary key (definition_name)
+);
+
+CREATE TABLE TASK_EXECUTION (
+  TASK_EXECUTION_ID BIGINT NOT NULL PRIMARY KEY,
+  START_TIME DATETIME DEFAULT NULL,
+  END_TIME DATETIME DEFAULT NULL,
+  TASK_NAME  VARCHAR(100),
+  EXIT_CODE INTEGER,
+  EXIT_MESSAGE VARCHAR(2500),
+  ERROR_MESSAGE VARCHAR(2500),
+  LAST_UPDATED TIMESTAMP,
+  EXTERNAL_EXECUTION_ID VARCHAR(255),
+  PARENT_EXECUTION_ID BIGINT
+);
+
+CREATE TABLE TASK_EXECUTION_PARAMS (
+  TASK_EXECUTION_ID BIGINT NOT NULL,
+  TASK_PARAM VARCHAR(2500),
+  constraint TASK_EXEC_PARAMS_FK foreign key (TASK_EXECUTION_ID)
+  references TASK_EXECUTION(TASK_EXECUTION_ID)
+);
+
+CREATE TABLE TASK_TASK_BATCH (
+  TASK_EXECUTION_ID BIGINT NOT NULL,
+  JOB_EXECUTION_ID BIGINT NOT NULL,
+  constraint TASK_EXEC_BATCH_FK foreign key (TASK_EXECUTION_ID)
+  references TASK_EXECUTION(TASK_EXECUTION_ID)
+);
+
+CREATE TABLE TASK_SEQ(
+  ID BIGINT NOT NULL,
+  UNIQUE_KEY CHAR(1) NOT NULL,
+  constraint UNIQUE_KEY_UN unique (UNIQUE_KEY)
+);
+
+INSERT INTO TASK_SEQ (ID, UNIQUE_KEY) select * from (select 0 as ID, '0' as UNIQUE_KEY) as tmp;
+
+CREATE TABLE TASK_LOCK (
+  LOCK_KEY CHAR(36),
+  REGION VARCHAR(100),
+  CLIENT_ID CHAR(36),
+  CREATED_DATE DATETIME(6) NOT NULL,
+  constraint LOCK_PK primary key (LOCK_KEY, REGION)
+);
+
+CREATE TABLE BATCH_JOB_INSTANCE (
+  JOB_INSTANCE_ID BIGINT  NOT NULL PRIMARY KEY,
+  VERSION BIGINT,
+  JOB_NAME VARCHAR(100) NOT NULL,
+  JOB_KEY VARCHAR(32) NOT NULL,
+  constraint JOB_INST_UN unique (JOB_NAME, JOB_KEY)
+);
```

Comment:
On the Spring batch definitions, it has `ENGINE=InnoDB` for each of the batch definitions.  Just curious why there is an omission here.   Possibly a CloudFoundry hiccup from days past?

Reply:
I think it is cruft, - that should be he default engine nowadays.

Reply:
We may want to ask @mminella about this.   I remember we had an issue in CF a while back and had to add this.

Reply:
I opted to leave those out as it's default and dba should choose default type anyway.